### ISSUE TITLE
Create PoC with generated code for the rialto parachain wrapped runtime

### ISSUE THREAD

**Issue by serban300** [paritytech/parity-bridges-common#1781]
Related to #446

Related to:

```
For (1) the solution would be to create transactions based on Metadata somehow or via RPC
```

I think it would be useful to create a PoC in which we use a runtime generated from metadata (with [subxt-codegen](https://docs.rs/subxt-codegen/latest/subxt_codegen)) for rialto parachain and see if it has obvious advantages over the minimalistic wrapped runtimes that we are using right now.

**Comment by serban300** [comment]
Opened a PoC for this: https://github.com/paritytech/parity-bridges-common/pull/1812

**Comment by serban300** [comment]
Resolved by #1812
